corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -490,6 +490,15 @@ def handle_pillow_error(pillow, change, exception):
         'exception_type': exception_path
     })
 
+    notify_exception(
+        None,
+        'Unexpected error in pillow',
```

Comment:
could be useful to put the exception name in the title to help with grouping in sentry